### PR TITLE
feat: added subscriptions in LPR

### PR DESCRIPTION
**Pull request by zamanafzal**
# For all changes
These changes are related to OPTIMIZELY Experiment. Will add the test cases if the experiment changes need to retain. Here is the ticket which will cover this.
https://2u-internal.atlassian.net/browse/ENT-6624

- [ ] Ensure adequate tests are in place (or reviewed existing tests cover changes)

# Only if submitting a visual change

- [ ] Ensure to attach screenshots
- [ ] Ensure to have UX team confirm screenshots

### CODE REVIEW

File: src/components/Admin/EmbeddedSubscription.jsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    match.params.subscriptionUUID = activeSubscriptions[0].uuid;
+  }
+  const inputChangeHandler = (event) => {
+    setSubscriptionUUID(event.target.value);
+    setFirstLoad(false);
+    match.params.subscriptionUUID = event.target.value;
+  };
+  return (
+    <div>{
+            !loading && activeSubscriptions.length > 0
```
Comment (saleem-latif) [inline src/components/Admin/EmbeddedSubscription.jsx:29]: Would it be helpful to show a Loading indicator if `loading==true` ?

File: src/components/Admin/licenses/LicenseManagementTable/index.jsx [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+    setSearchQuery,
+    setCurrentPage,
+    subscription,
+    users,
+    forceRefreshUsers,
+    loadingUsers,
+    setUserStatusFilter,
+  } = useContext(SubscriptionDetailContext);
+
+  // const refinedUsers = users?.filter(user =>  user.status !== REVOKED);
```
Comment (saleem-latif) [inline src/components/Admin/licenses/LicenseManagementTable/index.jsx:75]: nit: Remove this line.

**Review by saleem-latif — APPROVED** [review]
Just a couple of comments, otherwise looks good  👍

**Review by muhammad-ammar — APPROVED** [review]
LGTM.

**Pull request merged**